enhancements/ingress/set-delete-http-headers.md

```diff
@@ -791,6 +789,7 @@ The following table summarizes which headers are allowed to be customized via th
 | `strict-transport-security` | No                                                              | No                                                  | `haproxy.router.openshift.io/hsts_header` Route annotation                                                                                   |
 | `x-forwarded-for`           | Yes                                                             | Yes                                                 | `haproxy.router.openshift.io/set-forwarded-headers` Route annotation or IngressController `spec.httpHeaders.forwardedHeaderPolicy` API field |
 | `x-ssl`                     | Yes                                                             | Yes                                                 | No                                                                                                                                           |        
+| `cookie` or `set-cookie`      | No                                                              | No                                                  | No                                                                                                                                           |        
```

Comment:
We do have the `haproxy.router.openshift.io/disable_cookies` and `router.openshift.io/cookie_name` route annotations that you could list in this table.

Reply:
ok

Reply:
done